company-profiles/huubap.md

```diff
@@ -0,0 +1,29 @@
+# Huubap
+
+## Company blurb
+
+Huubap is a consulting agency.
+
+## Company size
+
+0-50.
+
+## Remote status
+
+Employees can work from anywhere in the world and they offer flexible working hours.
+
+## Region
+
+Asia
```

Comment:
"anywhere in the world" and "Asia" being your region are not compatible. 

Can you clarify what regions you are hiring in, please?